traffic_ops/traffic_ops_golang/cdn/namedelete.go

```diff
@@ -51,11 +53,12 @@ func DeleteName(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, errors.New("Failed to delete cdn name = "+string(cdnName)+" has delivery services or servers"), nil)
 		return
 	}
-	if err := deleteCDNByName(inf.Tx.Tx, tc.CDNName(cdnName)); err != nil {
+	if err := deleteCDNByName(inf.Tx.Tx, cdnName); err != nil {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("deleting CDN: "+err.Error()))
 		return
 	}
 	api.WriteRespAlert(w, r, tc.SuccessLevel, "cdn was deleted.")
+	api.CreateChangeLogRawTx(api.ApiChange, "CDN: "+string(cdnName)+", ID: "+strconv.Itoa(cdnID)+", ACTION: Deleted CDN", inf.User, inf.Tx.Tx)
```

Comment:
this doesn't seem to be working right. this is what it looks like to me when i delete a cdn

![image](https://user-images.githubusercontent.com/251272/62727254-48618000-b9d6-11e9-892e-8ec434a8e46f.png)

Reply:
Traffic Portal deletes CDNs by IDs, so that response is coming from the generic CRUD handler for audit logging rather than the route in `namedelete.go`